semcore/data-table/src/components/DataTable/DataTable.types.ts

```diff
@@ -128,6 +129,11 @@ export type DataTableProps<
      */
     renderEmptyData?: () => React.ReactNode;
 
+    /**
+     * For add some overlay over table cells.
```

Comment:
```suggestion
     * For adding an overlay over table cells.
```
(just a small grammar fix)